### DISCUSSION TITLE
Is there a difference between Subtitler using an external Translation service, vs the built-in Whisper translation?

### DISCUSSION THREAD

**Discussion by meekmeekmeek** [anupamkumar/subtitler#3] [Q&A] [answered]
I notice that if using the Whisper CLI directly, you can specify `--task translate` and it will translate internally; is it doing that via a built-in translation model, or is it also going to an external translation api like Google etc?

**Comment by anupamkumar** [comment, marked as answer]
Whisper's models are trained to only translate from any of their supported language to english only. For example, you cannot translate from Japanese to German using Whisper's built-in model. 

Whisper is strongest when used to transcribe audio to text in their supported languages. Google or DeepL are much much better at translating text than Whisper is. So, the utility uses external services to translate the transcribed text instead of relying on Whisper for doing this.
